cgi-bin/DW/Logic/ProfilePage.pm

```diff
@@ -838,6 +838,17 @@ sub external_services {
         };
     }
 
+    if ( my $emedium = $u->prop( 'medium' ) ) {
```

Comment:
This should be `my $medium = ...` reason being you need to have the raw contents so you can escape it for HTML later and for URL both.